conanfile.py

```diff
@@ -200,17 +210,24 @@ def get_configure_command_args(self):
         if not self.options.with_ldap:
             params.append("--disable-ldap")
 
-        if self.options.custom_cacert:
-            params.append('--with-ca-bundle=cacert.pem')
+        if self.options.with_ca_bundle == False:
+            params.append("--without-ca-bundle")
+        elif self.options.with_ca_bundle:
+            params.append(f"--with-ca-bundle={str(self.options.with_ca_bundle)}")
 
+        if self.options.with_ca_path == False:
+            params.append('--without-ca-path')
+        elif self.options.with_ca_path:
+            params.append(f"--with-ca-bundle={str(self.options.with_ca_bundle)}")
+
+        host = None
         # Cross building flags
-        if tools.cross_building(self.settings):
-            if self.settings.os == "Linux" and "arm" in self.settings.arch:
-                params.append('--host=%s' % self.get_linux_arm_host())
+        if tools.cross_building(self.settings) and self.settings.os in ["Linux", "iOS"]:
+            host = self.get_host()
 
-        return params
+        return params, host
 
-    def get_linux_arm_host(self):
+    def get_host(self):
         arch = None
         if self.settings.os == 'Linux':
```

Comment:
Check for arm should be moved here.

```suggestion
         if self.settings.os == 'Linux' and "arm" in self.settings.arch:
```

Reply:
I believe that won't work for 'aarch64-linux-gnu'.

Reply:
Why it will not work? Method `get_host` returns non-null value for arms and iOS and called for cross-compiling. It is the same logic as before the PR because the check
```python
if tools.cross_building(self.settings):
  if self.settings.os == "Linux" and "arm" in self.settings.arch:
    params.append('--host=%s' % self.get_linux_arm_host())
```
is moved inside the `get_host`.

Reply:
'aarch64' can be a user added value, if I believe the recipe comment. AFAIU the check before did not work since 'arm' does match 'aarch64'.

Reply:
Oh, I see!
So we can respect user choice by adding a last branch:
```python
        if self.settings.os == 'Linux':
            # aarch64 could be added by user
            if 'aarch64' in self.settings.arch:
                arch = 'aarch64-linux-gnu'
            elif 'arm' in self.settings.arch and 'hf' in self.settings.arch:
                arch = 'arm-linux-gnueabihf'
            elif 'arm' in self.settings.arch and self.arm_version(str(self.settings.arch)) > 4:
                arch = 'arm-linux-gnueabi'
            elif 'arm' in self.settings.arch:
                arch = 'arm-linux-gnu'
```

Reply:
lgtm. I have re-organize the ifs for clarirty.

Reply:
Yes, looks better now